mlrun/datastore/google_cloud_storage.py

```diff
@@ -45,6 +45,7 @@ def filesystem(self):
         self._filesystem = makeDatastoreSchemaSanitizer(
             filesystem_class,
             using_bucket=self.using_bucket,
+            use_listings_cache=False,
```

Comment:
Need to allow storage options to override this.

Reply:
Also, need to add a comment explaining why this is there, including the version this became necessary.